### PR TITLE
PR for Screens and Firebase Installation

### PR DESCRIPTION
**Pull request by manojasarsa**
This is PR includes:

- Screens using TypeScript
- Firebase Installation

### CODE REVIEW

**Review by samyakshah3008 — APPROVED** [review]
Clean code and UI 
Good implementation using firebase and typescript 
Good for approval

File: src/components/Header/Header.tsx [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+                        <div className="nav_center searchbar">
+                              <input type="search" className="input input_search search" placeholder="Search for quizzes" />
+                        </div>
+                        <div className="nav_right flex flex_justify_between flex_align_center">
+
+                              {/* <a className="badge_container badge_icon">
+                                    <i className="fas fa-user-circle profile"></i>
+                              </a> */}
+
+                              {/* TODO - Profile Options*/}
```
Comment (samyakshah3008) [inline src/components/Header/Header.tsx:17]: Can remove this code

File: src/components/QuesCard/quescard.css [hunk 2]
```diff
@@ -0,0 +1,98 @@
+.main_contain {
+    height: 100vh;
+    width: 100vw;
+    /* display: flex; */
```
Comment (samyakshah3008) [inline src/components/QuesCard/quescard.css]: Can remove this

**Review by samyakshah3008 — COMMENTED** [review]
Saw few parts in your code where there are unnecessary comments, you can remove them while refactoring

File: src/components/QuesCard/QuesCard.tsx [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+                        <form className="options flex flex_col flex_justify_center flex_align_center">
+                              <label>
+                                    <input type="radio" name="option" value="Option A" /> Option A
+                              </label>
+                              <label>
+                                    <input type="radio" name="option" value="Option B" /> Option B
+                              </label>
+                              <label>
+                                    <input type="radio" name="option" value="Option C" /> Option C
+                              </label>
```
Comment (znjs) [inline src/components/QuesCard/QuesCard.tsx:33]: can use map for the repetitive elements

File: src/components/QuesCard/quescard.css [hunk 4]
```diff
@@ -0,0 +1,98 @@
...
+}
+
+.ques_heading {
+    text-align: center;
+    margin: 2rem 0;
+}
+
+.ques_score_box {
+    /* display: flex;
+    justify-content: space-between; */
```
Comment (znjs) [inline src/components/QuesCard/quescard.css]: can remove commented code

File: src/components/QuesCard/quescard.css [hunk 5]
```diff
@@ -0,0 +1,98 @@
...
+
+.question {
+    padding: 1rem 0;
+    font-weight: var(--fw-semi-bold);
+}
+
+/* .options {
+    justify-content: center;
+    align-items: center;
+} */
```
Comment (znjs) [inline src/components/QuesCard/quescard.css]: can remove commented code

File: src/pages/Auth/auth.css [hunk 6]
```diff
@@ -0,0 +1,57 @@
...
+    margin-top: 5rem;
+    padding-bottom: 1rem;
+    background-color: var(--info-background-color);
+    box-shadow: none;
+}
+
+/* .input_container:hover {
+    opacity: 1;
+    transform: scale(1.10);
+} */
```
Comment (znjs) [inline src/pages/Auth/auth.css]: can remove commented code

File: src/pages/Auth/auth.css [hunk 7]
```diff
@@ -0,0 +1,57 @@
...
+}
+
+.logout_box .link {
+    color: var(--secondary-color);
+    font-size: 1.5rem;
+}
+
+/* .logout_box .link:hover {
+    color: var(--blue-color);
+} */
```
Comment (znjs) [inline src/pages/Auth/auth.css]: can remove commented code

File: src/pages/Home/home.css [hunk 8]
```diff
@@ -0,0 +1,210 @@
...
+}
+
+.ques_heading {
+      text-align: center;
+      margin: 2rem 0;
+}
+
+.ques_score_box {
+      /* display: flex;
+      justify-content: space-between; */
```
Comment (znjs) [inline src/pages/Home/home.css]: can remove commented code

File: src/pages/Home/home.css [hunk 9]
```diff
@@ -0,0 +1,210 @@
...
+
+.question {
+      padding: 1rem 0;
+      font-weight: var(--fw-semi-bold);
+}
+
+/* .options {
+      justify-content: center;
+      align-items: center;
+    } */
```
Comment (znjs) [inline src/pages/Home/home.css]: can remove commented code

File: src/pages/Profile/Profile.tsx [hunk 10]
```diff
@@ -0,0 +1,84 @@
...
+                        </div>
+
+                        <div className="quiz_profile_details flex flex_col flex_justify_start flex_align_center">
+
+                              <div className="card_container flex flex_row flex_align_between">
+                                    <h3 className="card_title">Achievements  <span className="quiz_points">8</span></h3>
+                                    <h3 className="card_title">Inventory  <span className="quiz_points">5</span></h3>
+                                    <h3 className="card_title">Quiz Wins  <span className="quiz_points">25</span></h3>
+                                    <h3 className="card_title">Highest Score  <span className="quiz_points">910</span></h3>
+                                    <h3 className="card_title">Correct Answers  <span className="quiz_points">218</span></h3>
```
Comment (znjs) [inline src/pages/Profile/Profile.tsx:35]: can make an array and use map here

File: src/pages/Result/Result.tsx [hunk 11]
```diff
@@ -0,0 +1,139 @@
...
+
+                <form className="options flex flex_col no_hover">
+                    <label>
+                        <input type="radio" name="option" value="Option A" /> Option A
+                    </label>
+                    <label className="correct">
+                        <input type="radio" name="option" value="Option B" /> Option B
+                    </label>
+                    <label className="wrong">
+                        <input type="radio" name="option" value="Option C" /> Option C
```
Comment (znjs) [inline src/pages/Result/Result.tsx:32]: can use map here for repetitive elements

File: src/pages/Result/Result.tsx [hunk 12]
```diff
@@ -0,0 +1,139 @@
...
+
+                <div className="ques_score_box">
+                    <p className="ques_num">Question 5</p>
+                </div>
+
+                <div className="question">
+                    <p>Lorem ipsum dolor sit amet consectetur adipisicing elit.
+                        Quidem aspernatur laudantium excepturi velit maiores quia non rem placeat ad esse?
+                    </p>
+                </div>
```
Comment (znjs) [inline src/pages/Result/Result.tsx:118]: can make a component of repetitive elements

File: src/pages/Rules/rules.css [hunk 13]
```diff
@@ -0,0 +1,48 @@
...
+    /* border-radius: 3rem; */
+    background-color: var(--info-background-color);
+    backdrop-filter: blur(4px);
+    border: 1px solid var(--primary-color);
+}
+
+/* .rules_box:hover {
+      opacity: 1;
+      transform: scale(1.09); 
+  }  */
```
Comment (znjs) [inline src/pages/Rules/rules.css]: can remove commented code

**Review by znjs — APPROVED** [review]
- Clean folder structure
- good naming conventions used 
- Good PR

**Pull request merged**